data/core/macros/traits.cfg

```diff
@@ -180,6 +181,7 @@ Quick is the most noticeable trait, particularly in slower moving units such as
             apply_to=hitpoints
             increase_total=-5%
         [/effect]
+        exclude_traits=intelligent
```

Comment:
I am not exactly sure why the core traits file is modified to ensure the combination `intelligent,quick` is never picked.